app/views/dashboards/show.html.erb

```diff
@@ -12,13 +12,26 @@
       text-align: center;
       margin: 50px;
     }
+    .imsa-logo {
+      position: fixed;
+      top: 10px;
+      left: 50px;
+      width: 100px;
+    }
+    .profile-icon {
+  position: fixed;
+  top: 10px;
+  right: 50px; 
+  width: 100px; 
+  cursor: pointer;
+}
 
     h2 {
       color: #333;
     }
 
     p {

```

Comment:
Could add better descriptive name instead of h2 and p so everyone knows which things are being moved